app/views/sessions/new.html.erb

```diff
@@ -1,25 +1,25 @@
 <!-- <h1>Sessions#new</h1>
 <p>Find me in app/views/sessions/new.html.erb</p> -->
 
-  <div class="container">
-    <div class="row">
-      <div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well">
-        <h2>Log In</h2>
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well">
+      <h2>Log In</h2>
 
-        <%= form_tag sessions_path do %>
-          <font size="3" color="red"><%= flash[:error] %></font>
-          <div class="field">
-            <%= text_field_tag :email, params[:email], {class: "form-control", placeholder: "Email"} %>
-          </div>
-          <br>
-          <div class="field">
-            <%= password_field_tag :password,"", {class: "form-control", placeholder: "Password"}  %>
-          </div>
-          <br>
-          <div class="actions">
-            <button class="btn btn-primary"> Log in </button>
-          </div>
-      <% end %>
-      </div>
+      <%= form_tag sessions_path do %>
+        <font size="3" color="red"><%= flash[:error] %></font>
```

Comment:
please don't use `<font>` in the `html` if possible, it's better to always keep all the styles in the css so we can easily update our site later